# AirBnB NY Locations Data Case Study

In this final project, you task will be to take the data provided and find evidance to answer the following questions.

1. Which hosts are the busiest and why?
2. How many neighborhood groups are available and which shows up the most?
3. Are private rooms the most popular in manhattan?
4. Which hosts are the busiest and based on their reviews?
5. Which neighorhood group has the highest average price?
6. Which neighborhood group has the highest total price?
7. Which top 5 hosts have the highest total price?
8. Who currently has no (zero) availability with a review count of 100 or more?
9. What host has the highest total of prices and where are they located?
10. When did Danielle from Queens last receive a review?

You will be given **4 hours** to complete this assignment. 
**Be Advised** I will go dark for this entire assignment time period. That said, any questions that you would like to ask about the data, or the project **MUST** be asked before the time starts. Once the time has started, I can no longer give information.

This is to similate what you will face when you are out in the wild. 

Happy Coding!

In [1]:
import pandas as pd
import numpy as np

In [4]:
df = pd.read_csv("AB_NYC_2019.csv")
df.head()

id                                              name  host_id  \
0  2539                Clean & quiet apt home by the park     2787   
1  2595                             Skylit Midtown Castle     2845   
2  3647               THE VILLAGE OF HARLEM....NEW YORK !     4632   
3  3831                   Cozy Entire Floor of Brownstone     4869   
4  5022  Entire Apt: Spacious Studio/Loft by central park     7192   

     host_name neighbourhood_group neighbourhood  latitude  longitude  \
0         John            Brooklyn    Kensington  40.64749  -73.97237   
1     Jennifer           Manhattan       Midtown  40.75362  -73.98377   
2    Elisabeth           Manhattan        Harlem  40.80902  -73.94190   
3  LisaRoxanne            Brooklyn  Clinton Hill  40.68514  -73.95976   
4        Laura           Manhattan   East Harlem  40.79851  -73.94399   

         room_type  price  minimum_nights  number_of_reviews last_review  \
0     Private room    149               1                  9  2018-10-19   
1  Entire home/apt    225               1                 45  2019-05-21   
2     Private room    150               3                  0         NaN   
3  Entire home/apt     89               1                270  2019-07-05   
4  Entire home/apt     80              10                  9  2018-11-19   

   reviews_per_month  calculated_host_listings_count  availability_365  
0               0.21                               6               365  
1               0.38                               2               355  
2                NaN                               1               365  
3               4.64                               1               194  
4               0.10                               1                 0

In [47]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 48895 entries, 0 to 48894
Data columns (total 16 columns):
 #   Column                          Non-Null Count  Dtype  
---  ------                          --------------  -----  
 0   id                              48895 non-null  int64  
 1   name                            48879 non-null  object 
 2   host_id                         48895 non-null  int64  
 3   host_name                       48874 non-null  object 
 4   neighbourhood_group             48895 non-null  object 
 5   neighbourhood                   48895 non-null  object 
 6   latitude                        48895 non-null  float64
 7   longitude                       48895 non-null  float64
 8   room_type                       48895 non-null  object 
 9   price                           48895 non-null  int64  
 10  minimum_nights                  48895 non-null  int64  
 11  number_of_reviews               48895 non-null  int64  
 12  last_review                     

In [6]:
# How many neighborhood groups are available and which shows up the most?
neighborhoods = df["neighbourhood_group"].value_counts()
neighborhoods

Manhattan        21661
Brooklyn         20104
Queens            5666
Bronx             1091
Staten Island      373
Name: neighbourhood_group, dtype: int64

#### There are 5 neighborhood groups 

In [7]:
# Are private rooms the most popular in manhattan?
manhattan = df[(df["neighbourhood_group"] == "Manhattan")]
manhattan


id                                               name    host_id  \
1          2595                              Skylit Midtown Castle       2845   
2          3647                THE VILLAGE OF HARLEM....NEW YORK !       4632   
4          5022   Entire Apt: Spacious Studio/Loft by central park       7192   
5          5099          Large Cozy 1 BR Apartment In Midtown East       7322   
7          5178                   Large Furnished Room Near B'way        8967   
...         ...                                                ...        ...   
48886  36483010                    Comfy 1 Bedroom in Midtown East  274311461   
48888  36484087  Spacious Room w/ Private Rooftop, Central loca...  274321313   
48892  36485431            Sunny Studio at Historical Neighborhood   23492952   
48893  36485609               43rd St. Time Square-cozy single bed   30985759   
48894  36487245  Trendy duplex in the very heart of Hell's Kitchen   68119814   

           host_name neighbourhood_group   neighbourhood  latitude  longitude  \
1           Jennifer           Manhattan         Midtown  40.75362  -73.98377   
2          Elisabeth           Manhattan          Harlem  40.80902  -73.94190   
4              Laura           Manhattan     East Harlem  40.79851  -73.94399   
5              Chris           Manhattan     Murray Hill  40.74767  -73.97500   
7           Shunichi           Manhattan  Hell's Kitchen  40.76489  -73.98493   
...              ...                 ...             ...       ...        ...   
48886          Scott           Manhattan         Midtown  40.75561  -73.96723   
48888            Kat           Manhattan  Hell's Kitchen  40.76392  -73.99183   
48892  Ilgar & Aysel           Manhattan          Harlem  40.81475  -73.94867   
48893            Taz           Manhattan  Hell's Kitchen  40.75751  -73.99112   
48894     Christophe           Manhattan  Hell's Kitchen  40.76404  -73.98933   

             room_type  price  minimum_nights  number_of_reviews last_review  \
1      Entire home/apt    225               1                 45  2019-05-21   
2         Private room    150               3                  0         NaN   
4      Entire home/apt     80              10                  9  2018-11-19   
5      Entire home/apt    200               3                 74  2019-06-22   
7         Private room     79               2                430  2019-06-24   
...                ...    ...             ...                ...         ...   
48886  Entire home/apt    200               6                  0         NaN   
48888     Private room    125               4                  0         NaN   
48892  Entire home/apt    115              10                  0         NaN   
48893      Shared room     55               1                  0         NaN   
48894     Private room     90               7                  0         NaN   

       reviews_per_month  calculated_host_listings_count  availability_365  
1                   0.38                               2               355  
2                    NaN                               1               365  
4                   0.10                               1                 0  
5                   0.59                               1               129  
7                   3.47                               1               220  
...                  ...                             ...               ...  
48886                NaN                               1               176  
48888                NaN                               1                31  
48892                NaN                               1                27  
48893                NaN                               6                 2  
48894                NaN                               1                23  

[21661 rows x 16 columns]

In [11]:
manhattan_rooms = df["room_type"].value_counts(normalize=True)
manhattan_rooms

Entire home/apt    0.519665
Private room       0.456611
Shared room        0.023724
Name: room_type, dtype: float64

#### Entire homes/apartments are the most popular in Manhattan at 52% of listings

In [16]:
# Which hosts are the busiest and based on their reviews?
review = df.sort_values('reviews_per_month',ascending=False)
review.head(5)

id                                               name    host_id  \
42075  32678719  Enjoy great views of the City in our Deluxe Room!  244361589   
42076  32678720           Great Room in the heart of Times Square!  244361589   
38870  30423106                 Lou's Palace-So much for so little  228415932   
27287  21550302  JFK Comfort.5 Mins from JFK Private Bedroom & ...  156684502   
28651  22176831      JFK 2 Comfort 5 Mins from JFK Private Bedroom  156684502   

      host_name neighbourhood_group        neighbourhood  latitude  longitude  \
42075   Row NYC           Manhattan     Theater District  40.75918  -73.98801   
42076   Row NYC           Manhattan     Theater District  40.75828  -73.98876   
38870    Louann              Queens             Rosedale  40.65417  -73.74158   
27287   Nalicia              Queens  Springfield Gardens  40.66110  -73.76830   
28651   Nalicia              Queens  Springfield Gardens  40.66158  -73.77050   

          room_type  price  minimum_nights  number_of_reviews last_review  \
42075  Private room    100               1                156  2019-07-07   
42076  Private room    199               1                 82  2019-07-07   
38870  Private room     45               1                 37  2019-07-08   
27287  Private room     80               1                403  2019-07-06   
28651  Private room     50               1                341  2019-07-08   

       reviews_per_month  calculated_host_listings_count  availability_365  
42075              58.50                               9               299  
42076              27.95                               9               299  
38870              20.94                               1               134  
27287              19.75                               3                26  
28651              17.82                               3                25

#### Row NYC, Louann, and Nalicia are very busy according to the amount of reviews per month

In [27]:
#Which neighorhood group has the highest average price?
neigh = df.groupby("neighbourhood_group")["price"].mean().sort_values(ascending=False)
neigh

neighbourhood_group
Manhattan        196.875814
Brooklyn         124.383207
Staten Island    114.812332
Queens            99.517649
Bronx             87.496792
Name: price, dtype: float64

#### Manhattan has the highest average price at $196 per night 

In [33]:
# Which neighbor hood group has the highest total price?
neigh_tot = df.groupby("neighbourhood_group")["price"].sum().sort_values(ascending=False)
#a = neigh_tot / neigh_tot
#a
neigh_tot

neighbourhood_group
Manhattan        4264527
Brooklyn         2500600
Queens            563867
Bronx              95459
Staten Island      42825
Name: price, dtype: int64

In [39]:
#Which top 5 hosts have the highest total price?
top_hosts = df.groupby("host_name")["price"].sum().sort_values(ascending=False)
top_hosts

host_name
Sonder (NYC)         82795
Blueground           70331
Michael              66895
David                65844
Alex                 52563
                     ...  
Carolann                12
Salim                   10
Vishanti & Jeremy       10
Qiuchi                   0
Aymeric                  0
Name: price, Length: 11452, dtype: int64

#### Hosts: Sonder (NYC), Blueground, Michael, David, Alex have the highest total prices total across all their listings 

In [42]:
num_prop = df.groupby("host_name")["id"].nunique().sort_values(ascending=False)
num_prop

host_name
Michael             417
David               403
Sonder (NYC)        327
John                294
Alex                279
                   ... 
Marilynn              1
Maris                 1
Marisa & Tom          1
Marisa And Colin      1
'Cil                  1
Name: id, Length: 11452, dtype: int64

In [49]:
b= top_hosts / num_prop
b.sort_values(ascending=False)


host_name
Olson                9999.0
Rum                  8500.0
Jay And Liz          6000.0
Sarah-B              5000.0
Sarah-2              5000.0
                      ...  
Carolann               12.0
Vishanti & Jeremy      10.0
Salim                  10.0
Qiuchi                  0.0
Aymeric                 0.0
Length: 11452, dtype: float64

#### Olson lists the highest listing at $9999

In [66]:
# Who currently has no (zero) availability with a review count of 100 or more?
review_100 = df[(df["number_of_reviews"] >= 100)]
review_100

id                                               name    host_id  \
3          3831                    Cozy Entire Floor of Brownstone       4869   
7          5178                   Large Furnished Room Near B'way        8967   
8          5203                 Cozy Clean Guest Room - Family Apt       7490   
9          5238                 Cute & Cozy Lower East Side 1 bdrm       7549   
11         5441                    Central Manhattan/near Broadway       7989   
...         ...                                                ...        ...   
33943  26877259       *Times Square Luxury Apartment Private Room*  172864847   
35014  27759146             Cozy corner near Empire State Building  209549523   
36238  28826608                                 “For Heaven Cakes”  217379941   
37093  29489893   Beautiful furnished private studio with backyard  222098649   
42075  32678719  Enjoy great views of the City in our Deluxe Room!  244361589   

         host_name neighbourhood_group        neighbourhood  latitude  \
3      LisaRoxanne            Brooklyn         Clinton Hill  40.68514   
7         Shunichi           Manhattan       Hell's Kitchen  40.76489   
8        MaryEllen           Manhattan      Upper West Side  40.80178   
9              Ben           Manhattan            Chinatown  40.71344   
11            Kate           Manhattan       Hell's Kitchen  40.76076   
...            ...                 ...                  ...       ...   
33943          Jae           Manhattan     Theater District  40.75964   
35014      Mariluz           Manhattan              Midtown  40.74858   
36238        Brent              Queens  Springfield Gardens  40.66457   
37093      Melissa              Queens              Jamaica  40.68547   
42075      Row NYC           Manhattan     Theater District  40.75918   

       longitude        room_type  price  minimum_nights  number_of_reviews  \
3      -73.95976  Entire home/apt     89               1                270   
7      -73.98493     Private room     79               2                430   
8      -73.96723     Private room     79               2                118   
9      -73.99037  Entire home/apt    150               1                160   
11     -73.98867     Private room     85               2                188   
...          ...              ...    ...             ...                ...   
33943  -73.98731     Private room    100               1                137   
35014  -73.98341      Shared room     62               1                112   
36238  -73.76918  Entire home/apt     75               1                132   
37093  -73.79063  Entire home/apt     20               1                111   
42075  -73.98801     Private room    100               1                156   

      last_review  reviews_per_month  calculated_host_listings_count  \
3      2019-07-05               4.64                               1   
7      2019-06-24               3.47                               1   
8      2017-07-21               0.99                               1   
9      2019-06-09               1.33                               4   
11     2019-06-23               1.50                               1   
...           ...                ...                             ...   
33943  2019-07-07              12.05                               1   
35014  2019-06-13              10.77                               3   
36238  2019-07-05              15.78                               1   
37093  2019-07-07              13.11                               1   
42075  2019-07-07              58.50                               9   

       availability_365  
3                   194  
7                   220  
8                     0  
9                   188  
11                   39  
...                 ...  
33943                36  
35014                 0  
36238                28  
37093                41  
42075               299  

[3044 rows x 16 columns]

In [76]:
no_avail_review_100  = review_100[(review_100["availability_365"] == 0)]
no_avail_review_100

id                                       name    host_id  \
8          5203         Cozy Clean Guest Room - Family Apt       7490   
94        20913         Charming 1 bed GR8 WBurg LOCATION!      79402   
132       30031           NYC artists’ loft with roof deck     129352   
174       44221             Financial District Luxury Loft     193722   
180       45556      Fort Greene, Brooklyn: Center Bedroom      67778   
...         ...                                        ...        ...   
29581  22705516         The Quietest Block in Manhattan :)  127740507   
30461  23574142                          queens get away!!  176185168   
31250  24267706  entire sunshine of the spotless mind room   21074914   
32670  25719044                COZY Room for Female Guests   40119874   
35014  27759146     Cozy corner near Empire State Building  209549523   

        host_name neighbourhood_group              neighbourhood  latitude  \
8       MaryEllen           Manhattan            Upper West Side  40.80178   
94     Christiana            Brooklyn               Williamsburg  40.70984   
132           Sol            Brooklyn                 Greenpoint  40.73494   
174         Coral           Manhattan         Financial District  40.70666   
180          Doug            Brooklyn                Fort Greene  40.68863   
...           ...                 ...                        ...       ...   
29581    Kathleen           Manhattan                     Harlem  40.83102   
30461       Janet              Queens                  Laurelton  40.68209   
31250      Albert            Brooklyn         Bedford-Stuyvesant  40.68234   
32670    Stephany            Brooklyn  Prospect-Lefferts Gardens  40.66242   
35014     Mariluz           Manhattan                    Midtown  40.74858   

       longitude        room_type  price  minimum_nights  number_of_reviews  \
8      -73.96723     Private room     79               2                118   
94     -73.95775  Entire home/apt    100               5                168   
132    -73.95030     Private room     50               3                193   
174    -74.01374  Entire home/apt    196               3                114   
180    -73.97691     Private room     65               2                206   
...          ...              ...    ...             ...                ...   
29581  -73.94181     Private room     65               2                103   
30461  -73.73662     Private room     65               1                119   
31250  -73.91318     Private room     49               1                102   
32670  -73.94417     Private room     48               1                131   
35014  -73.98341      Shared room     62               1                112   

      last_review  reviews_per_month  calculated_host_listings_count  \
8      2017-07-21               0.99                               1   
94     2018-07-22               1.57                               1   
132    2019-05-20               1.86                               1   
174    2019-06-20               1.06                               1   
180    2019-06-30               1.92                               2   
...           ...                ...                             ...   
29581  2019-07-07               5.89                               2   
30461  2018-12-24               7.79                               1   
31250  2019-07-05               6.73                               3   
32670  2019-05-31               9.97                               2   
35014  2019-06-13              10.77                               3   

       availability_365  
8                     0  
94                    0  
132                   0  
174                   0  
180                   0  
...                 ...  
29581                 0  
30461                 0  
31250                 0  
32670                 0  
35014                 0  

[162 rows x 16 columns]

In [77]:
no_avail_review_100_count  = review_100[(review_100["availability_365"] == 0)].nunique()
no_avail_review_100_count

id                                162
name                              162
host_id                           148
host_name                         141
neighbourhood_group                 5
neighbourhood                      50
latitude                          161
longitude                         162
room_type                           3
price                              84
minimum_nights                      9
number_of_reviews                  91
last_review                       114
reviews_per_month                 140
calculated_host_listings_count      7
availability_365                    1
dtype: int64

#### 162 listings have 0 availability with a review count of over 100; 148 unique hosts 

In [78]:
# What host has the highest total of prices and where are they located?
top_hosts = df.groupby("host_name")["price"].sum().sort_values(ascending=False)
top_hosts


host_name
Sonder (NYC)         82795
Blueground           70331
Michael              66895
David                65844
Alex                 52563
                     ...  
Carolann                12
Salim                   10
Vishanti & Jeremy       10
Qiuchi                   0
Aymeric                  0
Name: price, Length: 11452, dtype: int64

In [80]:
sonder = df[(df["host_name"]== "Sonder (NYC)")]
sonder

id                                               name    host_id  \
38293  30181691      Sonder | 180 Water | Incredible 2BR + Rooftop  219517861   
38294  30181945         Sonder | 180 Water | Premier 1BR + Rooftop  219517861   
38588  30347708        Sonder | 180 Water | Charming 1BR + Rooftop  219517861   
39769  30937590            Sonder | The Nash | Artsy 1BR + Rooftop  219517861   
39770  30937591        Sonder | The Nash | Lovely Studio + Rooftop  219517861   
...         ...                                                ...        ...   
47691  35871510  Sonder | 116 John | Vibrant Studio + Fitness Room  219517861   
47692  35871511     Sonder | 116 John | Vibrant 1BR + Fitness Room  219517861   
47693  35871515         Sonder | 116 John | Stunning 1BR + Rooftop  219517861   
47814  35936418          Sonder | 116 John | Polished Studio + Gym  219517861   
47821  35937891            Sonder | 116 John | Simple Studio + Gym  219517861   

          host_name neighbourhood_group       neighbourhood  latitude  \
38293  Sonder (NYC)           Manhattan  Financial District  40.70637   
38294  Sonder (NYC)           Manhattan  Financial District  40.70771   
38588  Sonder (NYC)           Manhattan  Financial District  40.70743   
39769  Sonder (NYC)           Manhattan         Murray Hill  40.74792   
39770  Sonder (NYC)           Manhattan         Murray Hill  40.74771   
...             ...                 ...                 ...       ...   
47691  Sonder (NYC)           Manhattan  Financial District  40.70818   
47692  Sonder (NYC)           Manhattan  Financial District  40.70691   
47693  Sonder (NYC)           Manhattan  Financial District  40.70772   
47814  Sonder (NYC)           Manhattan  Financial District  40.70840   
47821  Sonder (NYC)           Manhattan  Financial District  40.70707   

       longitude        room_type  price  minimum_nights  number_of_reviews  \
38293  -74.00645  Entire home/apt    302              29                  0   
38294  -74.00641  Entire home/apt    229              29                  1   
38588  -74.00443  Entire home/apt    232              29                  1   
39769  -73.97614  Entire home/apt    262               2                  8   
39770  -73.97528  Entire home/apt    255               2                 14   
...          ...              ...    ...             ...                ...   
47691  -74.00631  Entire home/apt    135              29                  0   
47692  -74.00682  Entire home/apt    165              29                  0   
47693  -74.00673  Entire home/apt    165              29                  0   
47814  -74.00518  Entire home/apt    699              29                  0   
47821  -74.00557  Entire home/apt    699              29                  0   

      last_review  reviews_per_month  calculated_host_listings_count  \
38293         NaN                NaN                             327   
38294  2019-05-29               0.73                             327   
38588  2019-05-21               0.60                             327   
39769  2019-06-09               1.86                             327   
39770  2019-06-10               2.59                             327   
...           ...                ...                             ...   
47691         NaN                NaN                             327   
47692         NaN                NaN                             327   
47693         NaN                NaN                             327   
47814         NaN                NaN                             327   
47821         NaN                NaN                             327   

       availability_365  
38293               309  
38294               219  
38588               159  
39769                91  
39770                81  
...                 ...  
47691               339  
47692               342  
47693               347  
47814               327  
47821               341  

[327 rows x 16 columns]

#### Sonder has 327 properties in Manhattan 

In [83]:
# When did Danielle from Queens last receive a review?
danielle = df[(df["host_name"]== "Danielle") & (df["neighbourhood_group"]=="Queens")]
danielle.sort_values("last_review")

id                                               name    host_id  \
27021  21386105  Quiet & clean 1br haven with balcony near the ...  154256662   
33861  26814763  One bedroom with full bed / 1 stop from Manhattan  201647469   
7086    5115372  Comfy Room Family Home LGA Airport NO CLEANING...   26432133   
20403  16276632  Cozy Room Family Home LGA Airport NO CLEANING FEE   26432133   
22068  17754072                Bed in Family Home Near LGA Airport   26432133   
21517  17222454   Sun Room Family Home LGA Airport NO CLEANING FEE   26432133   
22469  18173787  Cute Tiny Room Family Home by LGA NO CLEANING FEE   26432133   
16349  13151075                    ASTORIA APARTMENT OUTDOOR SPACE   18051286   

      host_name neighbourhood_group     neighbourhood  latitude  longitude  \
27021  Danielle              Queens           Astoria  40.77134  -73.92424   
33861  Danielle              Queens  Long Island City  40.74565  -73.94699   
7086   Danielle              Queens     East Elmhurst  40.76374  -73.87103   
20403  Danielle              Queens     East Elmhurst  40.76335  -73.87007   
22068  Danielle              Queens     East Elmhurst  40.76389  -73.87155   
21517  Danielle              Queens     East Elmhurst  40.76367  -73.87088   
22469  Danielle              Queens     East Elmhurst  40.76380  -73.87238   
16349  Danielle              Queens           Astoria  40.77221  -73.92901   

             room_type  price  minimum_nights  number_of_reviews last_review  \
27021  Entire home/apt    250               3                  1  2018-01-02   
33861     Private room    108               2                 13  2019-06-20   
7086      Private room     54               1                430  2019-07-03   
20403     Private room     48               1                510  2019-07-06   
22068      Shared room     38               1                224  2019-07-06   
21517     Private room     48               1                417  2019-07-07   
22469     Private room     48               1                436  2019-07-08   
16349     Private room     50               1                  0         NaN   

       reviews_per_month  calculated_host_listings_count  availability_365  
27021               0.05                               1               180  
33861               1.74                               1               333  
7086               13.45                               5               347  
20403              16.22                               5               341  
22068               7.96                               5                80  
21517              14.36                               5               338  
22469              16.03                               5               337  
16349                NaN                               1                 0

#### Danielle from Queens got her last review on 2019-07-08

## Further Questions

1. Which host has the most listings?

In [85]:
num_prop = df.groupby("host_name")["id"].nunique().sort_values(ascending=False)
num_prop

host_name
Michael             417
David               403
Sonder (NYC)        327
John                294
Alex                279
                   ... 
Marilynn              1
Maris                 1
Marisa & Tom          1
Marisa And Colin      1
'Cil                  1
Name: id, Length: 11452, dtype: int64

In [109]:
test = df.groupby("host_name")["calculated_host_listings_count"].count().sort_values(ascending=False)
test

host_name
Michael             417
David               403
Sonder (NYC)        327
John                294
Alex                279
                   ... 
Marilynn              1
Maris                 1
Marisa & Tom          1
Marisa And Colin      1
'Cil                  1
Name: calculated_host_listings_count, Length: 11452, dtype: int64

#### Michael has the most listings at 417

2. How many listings have completely open availability?

In [92]:
open_avai = (df["availability_365"]==0).value_counts()
open_avai

False    31362
True     17533
Name: availability_365, dtype: int64

#### 17,533 listings have completely open availablity 

3. What room_types have the highest review numbers?

In [106]:
rooms_most_reviews = df.groupby("room_type").count()
rooms_most_reviews

id   name  host_id  host_name  neighbourhood_group  \
room_type                                                                
Entire home/apt  25409  25402    25409      25400                25409   
Private room     22326  22318    22326      22314                22326   
Shared room       1160   1159     1160       1160                 1160   

                 neighbourhood  latitude  longitude  price  minimum_nights  \
room_type                                                                    
Entire home/apt          25409     25409      25409  25409           25409   
Private room             22326     22326      22326  22326           22326   
Shared room               1160      1160       1160   1160            1160   

                 number_of_reviews  last_review  reviews_per_month  \
room_type                                                            
Entire home/apt              25409        20332              20332   
Private room                 22326        17665              17665   
Shared room                   1160          846                846   

                 calculated_host_listings_count  availability_365  
room_type                                                          
Entire home/apt                           25409             25409  
Private room                              22326             22326  
Shared room                                1160              1160

#### Entire home and apartment have the highest amount of reviews

# Final Conclusion

In this cell, write your final conclusion for each of the questions asked.

Also, if you uncovered some more details that were not asked above, please describe them here.

-- I found it interesting that some hosts own so many listings! I bet they are suffering right now during the COVID-19 crisis. It will probably make some people reconsider buying investment properties for AirBnb. 